src/components/InputLabel/InputLabel.tsx

```diff
@@ -22,6 +33,14 @@ export const InputLabel: FC<InputLabelProps> = ({
     tooltip,
     bold,
 }) => {
+    const tooltips = [];
+
+    if (Array.isArray(tooltip)) {
+        tooltips.push(...tooltip);
+    } else if (tooltip) {
+        tooltips.push(tooltip);
+    }
```

Comment:
```suggestion
    const tooltips = Array.isArray(tooltip) ? tooltip : [tooltip];
```

Reply:
If tooltip is undefined this will throw an error since the tooltips are being mapped but the first item in the array will be undefined

Reply:
L33:
`tooltip = [],`
Then we should be fine

Reply:
And maybe we need to rename to `tooltips` but I leave the decision to the platform team